specifyweb/specify/views.py

```diff
@@ -558,6 +620,10 @@ def update_record(record: models.Model):
                             "new_record_data": {
                                 "type": "object",
                                 "description": "The new record data."
+                            },
+                            "bg": {
```

Comment:
call it `background` instead? saving few bytes not worth confusing people

Reply:
think of the bytes! LOL, probably a good idea, will fix